### PR TITLE
PT-582 | Initialize different catgories on edit event page

### PR DESCRIPTION
**Pull request by Sartonon**
## Description

- Initialize different catgories on edit event page

### Closes
**[PT-582](https://helsinkisolutionoffice.atlassian.net/browse/PT-582):**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -122,10 +122,10 @@
...
   "devDependencies": {
     "@apollo/react-testing": "^3.1.4",
-    "@graphql-codegen/cli": "^1.13.3",
-    "@graphql-codegen/typescript": "^1.13.3",
-    "@graphql-codegen/typescript-operations": "^1.13.3",
-    "@graphql-codegen/typescript-react-apollo": "^1.13.3",
+    "@graphql-codegen/cli": "1.13.3",
+    "@graphql-codegen/typescript": "1.13.3",
+    "@graphql-codegen/typescript-operations": "1.13.3",
+    "@graphql-codegen/typescript-react-apollo": "1.13.3",
```
Comment (Sartonon) [inline package.json:128]: If i add "^" back, it installs wrong version and codegen won't work with current script

Comment (jorilindell) [reply package.json:128]: Ok, Better to use strict version in that case. Maybe all codegen related libraries could be updated at some point.

**Review by jorilindell — APPROVED** [review]
Looks good

File: package.json (hunk 1 above)
Comment (Sartonon) [reply package.json:128]: yeah

**Pull request merged**